IPython/core/display.py

```diff
@@ -852,18 +852,24 @@ def _data_and_metadata(self):
     def _repr_json_(self):
         return self._data_and_metadata()
 
-_css_t = """$("head").append($("<link/>").attr({
-  rel:  "stylesheet",
-  type: "text/css",
-  href: "%s"
-}));
+_css_t = """var link = document.createElement("link");
+	link.ref = "stylesheet";
+	link.type = "text/css";
+	link.href = "%s";
+	document.head.appendChild(link);
 """
 
-_lib_t1 = """$.getScript("%s", function () {
-"""
-_lib_t2 = """});
+_lib_t1 = """var script = document.createElement("script");
```

Comment:
Currently $.getScript returns a 'thenable' which can be awaited when the script has been executed.

An alternative which could preserve this is:
new Promise((resolve, reject) => {
  const script = document.createElement("script");
  script.onload = resolve;
  script.onerror = reject;
  script.src = "%s";
  document.head.appendChild(script);
}).then(() => {
  // data
});

This is a 'nice to have' as it primarily helps ordering and error handling at higher levels, but would require using the result when this item is displayed.